### DISCUSSION TITLE
Getting 404 on serve (macos arm64)

### DISCUSSION THREAD

**Discussion by Jovan27** [pocketbase/pocketbase#2503] [Q&A]
I am first time trying to use Pocketbase but I am unable to get it to work.

When I run `./pocketbase serve` it looks like it started normally.

![image](https://github.com/pocketbase/pocketbase/assets/33517856/186b4e7b-7863-4b46-bcc8-a94c46baf813)

But when I open the link, I get a 404 response in the browser

![image](https://github.com/pocketbase/pocketbase/assets/33517856/2d84766f-f346-416a-ba1a-d071dc1fc8f3)

I tried a few different pocketbase versions and different browsers, but the issue persists.

**Comment by ganigeorgiev** [comment]
What is the address you are trying to access that resolves to the 404 error? Note that the Admin UI is located at `http://127.0.0.1:8090/_/`. The base `http://127.0.0.1:8090` is used for serving static files from the `pb_public` directory (if there is any).

Do you have a reverse proxy or something else that could interfere with the request?

**Reply by Jovan27** [reply]
I was trying to access the base URL, it's my fault 🤦‍♂️, accessing `/_` works.
